### PR TITLE
Add versioning and disable pyright bytes type promotion

### LINKED ISSUE #37: Disable pyright bytes type promotions
We should add `disableBytesTypePromotions = true` to pyright configuration in pyproject.toml (see https://github.com/openzim/warc2zim/commit/baafc7509312e94b4248cb19b56d652396d66577 for instance)

See [pyright documentation](https://github.com/microsoft/pyright/blob/main/docs/configuration.md) and [PEP 688](https://peps.python.org/pep-0688/#no-special-meaning-for-bytes). I don't know why the legacy behavior is the default Pyright configuration.

@rgaudin WDYT?

### LINKED ISSUE #38: Add versionning to our conventions
It begins to be hard for me to know which conventions of this bootstrap have been deployed to which scraper. And as time goes by, it will only become harder.

I think it is time to begin to version these conventions.

I don't know how to do it properly, since:
- the CHANGELOG.md and README.md are more linked to the demo `great_project` than to the conventions
- the conventions are a mix of pure code and some wiki pages (but most conventions are in the code)

I wonder if we should:
- move great_project to a subdirectory and hence create a CHANGELOG and a README on root folder more dedicated to explain what the repo is about
- move all wiki pages to the code repo so that they are versioned together with the code base and reviewed as well in PRs (and easier to find)

WDYT?

### PR DESCRIPTION
**Pull request by benoit74**
Fix #37 
Fix #38 

### Added

- Add versioning to the convention
- Mention the adherence to Python convention in README.md

### Changed

- Move wiki doc to repo for proper review / versioning
- Disable pyright bytes type promotions

NOTA: I've also pushed tags corresponding to versions mentioned in CHANGELOG.md ; I've intentionaly created many versions as possible so that it is easier for a project to specify "where he currently is and what is left to do" and for reviewers to check that everything is in line with the convention.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Mandatory for a versioned project. Must follow [Keep A Changelog](https://keepachangelog.com/en/1.1.0/)'s format.
+
+Ideally, entries in response to Github issues should mention name.
```
Comment (rgaudin) [inline docs/CHANGELOG.md:3]: Mention _them_, no ?

**Review by rgaudin — APPROVED** [review]
Good ; please apply minor suggestions first.

I assume we'll release as `1.0` once merged, right?

Also don't forget to remove the Wiki entries and disable Wiki in settings

File: docs/CHANGELOG.md (hunk 1 above)
Comment (benoit74) [reply docs/CHANGELOG.md:3]: I didn't changed a word 🤣
Fixed

File: docs/Home.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Please check our [[Policy]] first.
```
[the remark this answers is not in the document]
Comment (benoit74) [reply docs/Home.md:1]: Both fixed

**Pull request merged**